Wireframe/index.html

```diff
@@ -3,31 +3,92 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
     <title>Wireframe</title>
     <link rel="stylesheet" href="style.css" />
   </head>
+
   <body>
-    <header>
-      <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
-    </header>
     <main>
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <header>
+          <h1>Wireframe</h1>
+          <p>
+            The purpose of the wireframe is to create visual representation of a
+            website.
+          </p>
+          <img
+            src="https://cdn-images.visual-paradigm.com/handbooks/agile-handbook/wireframe/01-youtube-wireframe-example.png"
+            alt="wireframe"
+            width="200"
+          />
+
+          <p>
+            The wireframe also creates the layout of mobile app, or digital
+            product's structure, focusing on user experience and functionality
+            beforedetailed design and development. Wireframes act as a skeletal
+            framework, it shows the relationships of key elements like text,
+            buttons, images, and menus, without the distractions of color,
+            graphics, or content. They help designers visualize the user
+            interface, identify potential usability issues, and align on the
+            project's direction early on.
+          </p>
+        </header>
+        <a
+          href="https://balsamiq.com/learn/articles/what-are-wireframes/#:~:text=That's%20why%20wireframes%20exist.,nothing%20feels%20out%20of%20place."
+          >Read more</a
+        >
+      </article>
+
+      <article>
+        <img
+          src="https://www.computerhope.com/jargon/r/readme.png"
+          alt="Readme document"
+          width="10"
+        />
+
+        <h2>README file</h2>
+
         <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
+          README Files are the contents and structure of a folder and/or a
+          dataset it is used so that a researcher can locate the information
+          they need. The README file provides essential information about the
+          project, including its purpose, usage instructions, installation
+          details, and how to contribute to it. It serves as the introduction to
+          the project, as it help users understand what it does and how to get
+          started.
         </p>
-        <a href="">Read more</a>
+        <a
+          href=" https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes"
+        >
+          Read more</a
+        >
+      </article>
+
+      <article>
+        <img
+          src="https://gitbookdown.dallasdatascience.com/img/git_branch_merge.png"
+          width="300"
+          alt="The branch formation"
```

Comment:
Would it be obvious what "The branch formation" means to someone? Could you be more descriptive?